README.md

```diff
@@ -3,765 +3,222 @@
+    ```bash
+    make add-vault-token
+    ```
 
-```bash
-# ...
+5. Write the private key in the vault. This step is needed in order to be able to transit the issuer's state. To perform that action the given account has to be funded. For Mumbai network you can request some testing Matic [here](https://mumbaifaucet.com/).
 
-# See Section: Getting A Public URL
-ISSUER_SERVER_URL=<https://unique-forwaring-or-public-url.ngrok-free.app>
-# Defaults for Basic Auth in Base64 ("user-issuer:password-issuer" = "dXNlci1pc3N1ZXI6cGFzc3dvcmQtaXNzdWVy")
-# If you just want to get started, don't change these
-ISSUER_API_AUTH_USER=user-issuer
-ISSUER_API_AUTH_PASSWORD=password-issuer
-# !!!MUST BE SET or other steps will not work
-ISSUER_ETHEREUM_URL=<YOUR_RPC_PROVIDER_URI_ENDPOINT>
-```
+    ```bash
+    make private_key=<YOUR_WALLET_PRIVATE_KEY> add-private-key
+    ```
 
-> **NOTE:** In case the Vault was loaded multiple times and a fresh start is needed, the following will remove remnant data:
+----
+**Troubleshooting:**
 
-```bash
-# FROM: ./
+In order to **stop** and **delete** all the containers.
 
-make clean-vault;
-# (Equivalent)
-#   rm -R infrastructure/local/.vault/data/init.out
-#   rm -R infrastructure/local/.vault/file/core/
-#   rm -R infrastructure/local/.vault/file/logical/
-#   rm -R infrastructure/local/.vault/file/sys/
+> [!WARNING]
+> This will permanently delete all data, making it necessary to create an Issuer DID again.
 
-# Expected Output/Prompt:
-#   rm -R infrastructure/local/.vault/data/init.out
-#   rm -R infrastructure/local/.vault/file/core/
-#   rm -R infrastructure/local/.vault/file/logical/
-#   rm -R infrastructure/local/.vault/file/sys/
+``` bash
+make down
 ```
 
-#### Start Redis Postgres & Vault
-
-This will start the necessary local services needed to store the wallet private key to the Hashicorp vault and allow storing data associated to the issuer.
+If you experience **problems** with the **vault**, follow these commands:
 
-```bash
-# FROM: ./
-
-make up;
-# (Equivalent)
-#   docker compose -p issuer -f ./infrastructure/local/docker-compose-infra.yml up -d redis postgres vault;
-
-# Expected Output:
-#   docker compose -p issuer -f /Users/username/path/to/sh-id-platform/infrastructure/local/docker-compose-infra.yml up -d redis postgres vault
-#   [+] Running 4/4
-#   ⠿ Network issuer-network       Created                                                                                   0.0s
-#   ⠿ Container issuer-vault-1     Started                                                                                   0.5s
-#   ⠿ Container issuer-redis-1     Started                                                                                   0.4s
-#   ⠿ Container issuer-postgres-1  Started  
+``` bash
+docker stop issuer-vault-1
+docker rm issuer-vault-1
+make clean-vault
+make up
+make add-vault-token
+make private_key=<YOUR_WALLET_PRIVATE_KEY> add-private-key
```

Comment:
I think these commands could be more didactic with some comments. Also, I'd consider splitting them in 2 blocks, since we need to leave some time for the stack to boot before adding the token to the env.

```bash
docker stop issuer-vault-1    // Stops the container issuer-vault-1 
docker rm issuer-vault-1      // Removes container issuer-vault-1
make clean-vault              // Removes all the data in the vault, including the token
make up                       // Starts the database, cache and vault storage (i.e, postgres, redis and vault)
```

Wait 20 secs so the vault can boot and generate a token.

```bash
make add-vault-token                                          // Adds the generated token to the ISSUER_KEY_STORE_TOKEN var in .env-issuer
make private_key=<YOUR_WALLET_PRIVATE_KEY> add-private-key    // Stores the private key in the vault
```

Reply:
I'm missing the comments in the second block.

Reply:
done